content/vendor/additional-objects/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+date: "2019-11-27T00:00:00Z"
+lastmod: "2019-11-27T00:00:00Z"
+title: "Additional Kubernetes Objects"
+aliases: [/vendor/additional-objects]
+weight: "2"
+hideFromList: true
+---
+Elements of a KOTS application lifecycle are controlled by various Custom Resource Definitions included in a KOTS application spec.  
+
+These additional Kubernetes Objects serve as metadata for kots and kotsadm, and don't require any CRD controller code to be deployed to the target cluster.
```

Comment:
Object should not be capitalized.

metadata for ~kots and kotsadm~. How about “metadata for the KOTS application that can be used by the Admin Console”

The message that they don’t require CRD controller code is accurate but I think we should dig into this more. That doesn’t help me understand how it does work (the kubectl plugins can use them), and actually that there isn’t a controller even created for these. I think the message to convey here is that these custom resources (instead of Kubernetes Objects) are removed before deploying and do not have any prerequisites or require any elevated permissions to install to an existing cluster.

Reply:
Right on.  How about this to replace the second section: "These additional Kubernetes objects serve as metadata for the KOTS application that can be used by the Admin Console, and by kubectl plug-ins on the client side.  There is no CRD controller code required, and thus no need for elevated permissions or installation of any additional components to the target cluster."